tools/keycloak/sync/SyncFactory.cs

```diff
@@ -5,43 +5,46 @@
 using System.Net.Http;
```

Comment:
@asanchezr @FuriousLlama this is the class that needs the most inspection. The workflow is:
1) Adding all claims from PIMS to keycloak as keycloak roles
2) Delete any claims from PIMS that are in keycloak but not in PIMS
3) Add all roles from PIMS to keycloak as keycloak composite roles
4) Delete any roles from PIMS that are in keycloak but not in PIMS
5) For every user in PIMS, ensure that their user role mappings are the same as they are in keycloak.